mkosi/run.py

```diff
@@ -11,7 +11,7 @@
 import traceback
 from pathlib import Path
 from types import TracebackType
-from typing import Any, Callable, Iterable, Mapping, Optional, Sequence, Type, TypeVar
```

Comment:
That ended up in the wrong commit during a rebase. It should be in *Print cmdline in run() before doing fspath() conversion* but ended up in the one afterwards. Not much of an issue, but could trip up bisecting.